librecad/src/lib/actions/rs_snapper.cpp

```diff
@@ -758,22 +771,56 @@ void RS_Snapper::drawSnapper()
                 }
                 else //orthogonal crosshair
                 {
+					double snapX=graphicView->toGuiX(pImpData->snapCoord.x);
+					double snapY=graphicView->toGuiY(pImpData->snapCoord.y);
+
+					double viewWidth=double(graphicView->getWidth());
```

Comment:
Technically, this does not actually originate from me, but from the original line 765, as you can see below when I was factoring out the expression. Do you want me to push another commit to change this?